portality/api/v2/crud/applications.py

```diff
@@ -58,6 +61,20 @@ def create(cls, data, account, dry_run=False):
             ia = IncomingApplication(data)
         except seamless.SeamlessException as e:
             raise Api400Error(str(e))
+        except dataobj.ScriptTagFoundException as e:
+            jdata = json.dumps(data, indent=4)
```

Comment:
does the email sending need to be in a try/catch so we don't lose the response if the email doesn't send?